README.md

```diff
@@ -60,13 +60,23 @@ $ vim templates/logsearch-for-cf.example-with-uaa-auth.yml
 $ scripts/generate_deployment_manifest ~/workspace/logsearch.yml templates/logsearch-for-cf.example-with-uaa-auth.yml > ~/workspace/logsearch-with-logsearch-for-cf.yml
 ```
 
-### 3. Update the logsearch deployment with the new manifest
+### 4. Update the logsearch deployment with the new manifest
 
 ```sh
 $ bosh deployment ~/workspace/logsearch-with-logsearch-for-cf.yml
 $ bosh deploy
 ```
 
+### 5. Update Cloud Foundry deployment to forward component logs to ingestor
+
+```yaml
+properties:
+  syslog_daemon_config:
+    address: <ingestor-static-ip>
+    port: 5515
```

Comment:
@jmcarp Default port for relp protocol 2514, here the reference from logsearch-boshrelease [ingestor_relp/spec](https://github.com/logsearch/logsearch-boshrelease/blob/develop/jobs/ingestor_relp/spec#L39), this may cause difficulties for people who are not familiar with the ELK 😅